File: Foundation/Scratch/Getting Started/c++/l003pythagorosTriplets.cpp
#include <iostream>
using namespace std;

int main(int argc, char **argv){
    int a,b,c;
    cin >> a >> b >> c;
    
    //write your code here
    bool ans =  (a*a == b*b + c*c || b*b == c*c + a*a ||c*c == a*a + b*b )?1:0;
    cout<<(boolalpha)<<ans<<endl;
    return 0;
    
}

File: Foundation/Scratch/Getting Started/java/L003PythagorosTriplets.java
public class L003PythagorosTriplets{

public static void main(String[] args){
    int a = scn.nextInt();
    int b = scn.nextInt();
    int c = scn.nextInt();
    
    
    //write your code here
    boolean ans =  (a*a == b*b + c*c || b*b == c*c + a*a ||c*c == a*a + b*b )?1:0;
    System.out.println(ans);
    
}
}
